### ISSUE TITLE
Feature Request: list notes

### ISSUE THREAD

**Issue by dwe3k** [dcchambers/note-keeper#54]
If I am using Organize by Date and don't create a note every day, it can be hard to remember what note files exist. I would like (created for myself) an `-l/--list` option to list all note files. Thus I can see the filename to be used with `-e/--edit` or `-d/--delete` options. My suggestion/function is:

```
list_notes() {
    for filename in $(ls -1 -R $NOTE_DIR | grep ".md"); do
        basename $filename .md
    done
}
```

This should find all of the Markdown note files in the Notes directory.

**Comment by dcchambers** [comment]
Good recommendation @dwe3k - I'll try to get this implemented this week.

**Comment by dcchambers** [comment]
@dwe3k Implemented and released in [v1.2](https://github.com/dcchambers/note-keeper/releases/tag/v1.2) - thanks for the suggestion!
